test/e2e/prometheus_test.go

```diff
@@ -84,3 +85,98 @@ func TestPrometheusVolumeClaim(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestPrometheusAlertmanagerAntiAffinity(t *testing.T) {
+	pods, err := f.KubeClient.CoreV1().Pods(f.Ns).List(metav1.ListOptions{FieldSelector: "status.phase=Running"})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	var alm = `affinity:
+	   podAntiAffinity:
+	      preferredDuringSchedulingIgnoredDuringExecution:
+	      - podAffinityTerm:
+	          labelSelector:
+	            matchExpressions:
+	            - key: alertmanager
+	              operator: In
+	              values:
+	              - main`
+
+	var k8s = `affinity:
+	   podAntiAffinity:
+	      preferredDuringSchedulingIgnoredDuringExecution:
+	      - podAffinityTerm:
+	          labelSelector:
+	            matchExpressions:
+	            - key: prometheus
+	              operator: In
+	              values:
+	              - k8s`
+
+	var testPod1 = "alertmanager-main"
+	var testPod2 = "prometheus-k8s"
+	var testNameSpace = "openshift-monitoring"
+
+	mapPodAntiAff := map[int]string{0: ""}
+	for idx, line := range strings.Split(alm, "\n") {
+		mapPodAntiAff[idx] = line
+	}
+	mapPodK8s := map[int]string{0: ""}
+	for idx, line := range strings.Split(k8s, "\n") {
+		mapPodK8s[idx] = line
+	}
+	
+	var resultCheckAlertmanagerMain = false
+	var resultCheckPrometheuseK8s = false
```

Comment:
please use shorter variable names to be more idiomatic.

Reply:
fixed